packages/contracts/deployments/README.md

```diff
@@ -31,9 +34,9 @@
 
 Network : __undefined (chain id: 69)__
 
-| Contract           | Address                                                                                                                         |
-| ------------------ | ------------------------------------------------------------------------------------------------------------------------------- |
-| OVM_GasPriceOracle | [0x038a8825A3C3B0c08d52Cc76E5E361953Cf6Dc76](https://undefined.etherscan.io/address/0x038a8825A3C3B0c08d52Cc76E5E361953Cf6Dc76) |
+|Contract|Address|
+|--|--|
+|OVM_GasPriceOracle|[0x038a8825A3C3B0c08d52Cc76E5E361953Cf6Dc76](https://undefined.etherscan.io/address/0x038a8825A3C3B0c08d52Cc76E5E361953Cf6Dc76)|
```

Comment:
Hmm I think this contract might be unused now. But that's out of scope for this PR.